src/pages/organize/[orgId]/campaigns/[campId]/index.tsx

```diff
@@ -207,78 +207,23 @@ const CampaignSummaryPage: PageWithLayout<CampaignCalendarPageProps> = ({ orgId,
-                    <Box border={ 1 } m={ 1 } p={ 3 } pb={ 5 }>
-                        <Typography component="h3" variant="h6">
-                            <Msg id="pages.organizeCampaigns.mobilization.heading" />
-                        </Typography>
-                        <Box alignItems="center" display="flex" flexDirection="column">
-                            <Phone style={{ color:grey[500], fontSize: 100  }}/>
-                            <Box p={ 2 } textAlign="center" width={ 0.5 }>
-                                <Typography color="textSecondary" variant="body1">
-                                    <Msg id="pages.organizeCampaigns.mobilization.copy" />
-                                </Typography>
-                            </Box>
-                            <NextLink href={ `/organize/${orgId}/campaigns/${campId}/call-assignments/new` } passHref>
-                                <Button color="primary" disabled={ true } variant="contained">
-                                    <Msg id="pages.organizeCampaigns.mobilization.create" />
-                                </Button>
-                            </NextLink>
-                        </Box>
-                    </Box>
-                    { /* Feedback and Surveys */ }
-                    <Box border={ 1 } m={ 1 } p={ 3 } pb={ 5 }>
-                        <Typography component="h3" variant="h6">
-                            <Msg id="pages.organizeCampaigns.feedback.heading" />
-                        </Typography>
-                        <Box alignItems="center" display="flex" flexDirection="column">
-                            <PlaylistAddCheck style={{ color:grey[500], fontSize: 100  }}/>
-                            <Box p={ 2 } textAlign="center" width={ 0.5 }>
-                                <Typography color="textSecondary" variant="body1">
-                                    <Msg id="pages.organizeCampaigns.feedback.copy" />
-                                </Typography>
-                            </Box>
-                            <NextLink href={ `/organize/${orgId}/campaigns/${campId}/surveys/new` } passHref>
-                                <Button color="primary" disabled={ true } variant="contained">
-                                    <Msg id="pages.organizeCampaigns.feedback.create" />
-                                </Button>
-                            </NextLink>
-                        </Box>
-                    </Box>
-                </Box>
+                    <Grid item md={ 6 } sm={ 12 } xs={ 12 }>
+                        <ZetkinSection title={ intl.formatMessage({ id: 'pages.organizeCampaigns.tasks' }) }>
+                            <TaskList hrefBase={ `/organize/${orgId}/campaigns/${campId}` } tasks={ tasks ?? [] } />
+                        </ZetkinSection>
+                    </Grid>
+                </Grid>
+                <ZetkinSpeedDial actions={ [ACTIONS.CREATE_EVENT, ACTIONS.CREATE_TASK] }/>
```

Comment:
now this page has the speed dial twice, causing a bug, when you click on an action, it takes two clicks to close

Reply:
Good catch, merge conflict issue from reverting some old code. Removed!